cmd/activator/request_log.go

```diff
@@ -29,7 +30,12 @@ import (
 
 func updateRequestLogFromConfigMap(logger *zap.SugaredLogger, h *pkghttp.RequestLogHandler) func(configMap *corev1.ConfigMap) {
 	return func(configMap *corev1.ConfigMap) {
-		newTemplate := configMap.Data["logging.request-log-template"]
+		obsconfig, err := metrics.NewObservabilityConfigFromConfigMap(configMap)
+		if err != nil {
+			logger.Errorw("Failed to get observability configmap.", zap.Error(err), "configmap", configMap)
+			return
+		}
```

Comment:
total nit, but nicer to have a newline after a guard clause I think

```suggestion
		}
		
```